src/components/shared/renderers/ListItem.jsx

```diff
@@ -0,0 +1,20 @@
+//@flow
+import React from 'react';
+import {listItem} from 'react-markdown/lib/renderers';
+import Checkbox from './Checkbox';
+
+type Props = {
+  checked?: boolean,
+  children: any,
+  onChangeCheckbox: (key: string, checkbox: boolean) => void,
+};
+
+const ListItem = (props: Props) => {
+  if (props.checked === null || props.checked === undefined) {
```

Comment:
I wonder if the guard should be checking if it is a checkbox item and returning `<Checkbox... />` if it is. Then the default return of this function will closer match it's typical flow and match the name.